PublicMapping/districtbuilder-classic
# Add Travis badge

## Overview

This PR adds a Travis badge for the `develop` branch build status

### Checklist


### Demo

_nice_

![image](https://user-images.githubusercontent.com/5702984/35876316-e2c1bd32-0b3f-11e8-8db1-5d5b711802e6.png)

## Testing Instructions

 * Go to [this branch](https://github.com/PublicMapping/DistrictBuilder/tree/feature/js/add-travis-badge) in the GitHub code view
 * Ch-ch-ch-check it out, there's a badge
